cmd/notary-server/main.go

```diff
@@ -94,6 +95,59 @@ func grpcTLS(configuration *viper.Viper) (*tls.Config, error) {
 	return tlsConfig, nil
 }
 
+// parses the configuration and determines which trust service and key algorithm
+// to return
+func getTrustService(configuration *viper.Viper,
+	signerFactory func(string, string, *tls.Config) *client.NotarySigner,
```

Comment:
Declare a `signerFactory` type and provide names for the arguments. Were I writing my own signer, it's not clear from looking at this function signature what the first 2 arguments are.

```
type signerFactory func(host string, port string, tlsConf *tls.Config) *client.NotarySigner
```